README.md

```diff
@@ -138,14 +147,15 @@ The main use-case is for JWK where your public key is in a JSON format like so:
    "n":"yRE6rHuNR0QbHO3H3Kt2pOKGVhQqGZXInOduQNxXzuKlvQTLUTv4l4sggh5_CYYi_cvI-SXVT9kPWSKXxJXBXd_4LkvcPuUakBoAkfh-eiFVMh2VrUyWyj3MFl0HTVF9KwRXLAcwkREiS3npThHRyIxuy0ZMeZfxVL5arMhw1SRELB8HoGfG_AtH89BIE9jDBHZ9dLelK9a184zAf8LwoPLxvJb3Il5nncqPcSfKDDodMFBIMc4lQzDKL5gvmiXLXB1AGLm8KBjfE8s3L5xqi-yUod-j8MtvIj812dkS4QMiRVN_by2h3ZY8LYVGrqZXZTcgn2ujn8uKjXLZVD5TdQ"
 }
 ```
+The JWK values can usually be found by looking up the OIDC configuration of the token issuer:
 
-```rust
-// `token` is a struct with 2 fields: `header` and `claims` where `claims` is your own struct.
-let token = decode::<Claims>(&token, &DecodingKey::from_rsa_components(jwk["n"], jwk["e"]), &Validation::new(Algorithm::RS256))?;
-```
+1. append `/.well-known/openid-configuration` to the value of `iss` field from the token and download the JSON file from that URL
+2. look up the URL for `jwks_uri` field in the downloaded document
+3. download JWKS (a JWK set) from `jwks_uri` and use the record with `kid` matching `kid` in the token's header
 
-If your key is in PEM format, it is better performance wise to generate the `DecodingKey` once in a `lazy_static` or
-something similar and reuse it.
+If you are the token issuer, you should be able to find the link to your JWKS somewhere in your token provider's control panel.
```

Comment:
Honestly I'm not sure this part on JWKs should be in the docs. For example, it would be different if you use JWKs from Auth0/Hydra so that would be just confusing many users.